test/runtests.jl

```diff
@@ -22,13 +22,14 @@ using RDatasets
     x = [13.0,14.0,14.35,15.0,16.0]
     y = [0.369486,  0.355579, 0.3545, 0.356952, 0.36883]
     model = loess(x,y)
-    @test Loess.predict(model,x) ≈ y
+    @test predict(model,x) ≈ y
```

Comment:
```suggestion
    @test fitted(model) ≈ y
```
Since there's already a test for `predict`, you can turn this one into a test for `fitted` since there are no others for it